android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -607,7 +823,6 @@ private void startPlayback() {
                 case Player.STATE_IDLE:
                 case Player.STATE_ENDED:
                     initializePlayer();
-                    break;
```

Comment:
are you sure you need to remove this break ?

Reply:
Looks like some merge conflict issue. Great catch! I added it back.